### ISSUE TITLE
FCAN fire should be yellow

### ISSUE THREAD

**Issue by mc776** [freedoom/freedoom#850]
So after the torch update I started poking around to see what other flame effects could use some improvement.

The way the FCAN flames always tilt to the right have bothered me for a long time even in Doom and I wanted to see what I could do there.

While discussing the issue on Discord someone pointed out that a lot of maps have burning barrels placed behind level geometry that hides the barrel, using it as a generic "stuff is on fire" sprite.

Sometimes to the point of re-inventing Doom's burning drum:
![_](https://user-images.githubusercontent.com/24984517/209416169-42d86b0d-1eb8-44c8-abce-6cf9f0124482.jpg)

Most random things on the street that catch fire don't emit a pure blue flame.

That particular usage on Map16 is also a motivation for straightening out the flame, because right now a good chunk of it visibly clips into the sector barrel/pipe.

Making this into a yellow flame that goes straight up would not make the sprite more or less Doom-like than it currently is - different shape same colour rather than the other way around.

So I can cook up a revised barrel that does this, but before I do that I just want to check - was there any reason why we need to keep the blue flame *or* the tilt?

**Comment by mc776** [comment]
What I'm cooking up, with current and Doom for comparison:
![image](https://user-images.githubusercontent.com/24984517/209494165-782c796e-3424-4209-9b30-8d38c75530c3.png)

**Comment by mc776** [comment]
![sample](https://user-images.githubusercontent.com/24984517/210160842-152a936e-a6a7-4721-8955-e05e3f4af9bf.png)

The above version always bothered me because it always looked to me like *a sign indicating fire* rather than an actual fire.

Hopefully this messier version works better but let's see if anyone vociferously prefers the cleaner-looking version first.

**Comment by Talon1024** [comment]
I think I prefer the cleaner-looking version, and I don't think making it messier is necessarily the right way to make it look like fire. Try different shapes instead of just a triangle.

**Comment by mc776** [comment]
Now that I've slept on it I'm pretty sure the newer version is better.

It's not the triangular shape that's the problem, but the fact that we've only got 3 frames.

With the torches we had 4 frames which could allow for a fairly smooth cycle, combined with the single point where the flame is coming from allowing the use of a single main tongue for constant movement. 

With the wider surface area, the big problem is that a real fire would not keep a consistent basic shape - though on average it should more or less be biased in favour of the flame being taller near the middle.

Reducing horizontal movement helps with the limited frames (which was how I got this new version), but it would still be very visible 3 distinct waggling tongues of flame rather than a contiguous fire coming out of a wide barrel mouth - making any drastic change in any of which would stick out a lot and make the animation look like a slide show.

The new version is basically the old resized and copypasted 3 times so we've got 9 tongues going at once. Some of these go through exactly the sort of drastic change that I couldn't do with 3, because the surrounding less-drastic movement makes up for it.
